boards/nxp/imx93_evk/board.c

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2024 NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/init.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/drivers/gpio.h>
+
+LOG_MODULE_REGISTER(board_control, CONFIG_BOARD_MIMX93_EVK_A55_LOG_LEVEL);
+
+#if DT_HAS_COMPAT_STATUS_OKAY(imx93evk_exp_sel)
```

Comment:
if no imx93evk-exp-sel device node available,  we could also call board_init_exp_sel to set exp_sel to a default value, such as 'A", otherwise, exp_sel can be configured by dts node property

Reply:
To avoid pulling unnecessary code to project (and some basic zephyr tests) that dose not use EXP_SEL, I've added `CONFIG_BOARD_MIMX93_EVK_EXP_SEL_INIT` option.